trustchain-ion/src/verifier.rs

```diff
@@ -296,6 +296,18 @@ where
         }
         Ok(self.bundles.lock().unwrap().get(did).cloned().unwrap())
     }
+    /// Resolves the given DID to obtain the DID Document and Document Metadata.
```

Comment:
Moved the `resolve_did` method of `IONVerifier` to `FullClient` impl since resolution from the verifier on `LightClient` is not required (HTTP endpoint used instead).